src/components/NavBar/NavBar.css

```diff
@@ -1,90 +1,83 @@
 /* Position and size our burger button */
 .bm-burger-button {
-    position: fixed;
-    width: 36px;
-    height: 36px;
-    right: 32px;
-    top: 32px;
+  position: fixed;
+  width: 36px;
+  height: 36px;
+  right: 32px;
+  top: 32px;
 }
 
 /* Color to our burger */
 .bm-burger-bars {
-    background: white;
+  background: white;
 }
 
 /* Color to our burger icon on hover */
 .bm-burger-bars-hover {
-    background: rgb(255, 255, 255);
+  background: rgb(255, 255, 255);
 }
 
 /* Size of the cross button */
 .bm-cross-button {
-    height: 24px;
-    width: 24px;
+  height: 24px;
+  width: 24px;
 }
 
 /* Color of cross button */
 .bm-cross {
-    background: #fff;
+  background: #fff;
 }
 
 /* Sidebar wrapper styles */
 .bm-menu-wrap {
-    position: fixed;
-    height: 100vh;
-    overflow-x: hidden;
-    box-shadow: 0px 4px 31px rgba(202, 59, 59, 0.15);
+  position: fixed;
+  height: 100vh;
+  overflow-x: hidden;
+  box-shadow: 0px 4px 31px rgba(202, 59, 59, 0.15);
+  background-color: red;
```

Comment:
This red color doesn't look good. also, is this required to add?

Reply:
yes i was just playing around with code for a min, I forgot to delete that background color: red statement.